packages/dnsmap/PKGBUILD

```
@@ -2,30 +2,43 @@
 # See COPYING for license details.
 
 pkgname=dnsmap
-pkgver=0.30
-pkgrel=4
-groups=('blackarch' 'blackarch-fingerprint')
+pkgver=0.36.r6.g96b17ad
+pkgrel=1
 pkgdesc='Passive DNS network mapper.'
 arch=('x86_64' 'aarch64')
-url='http://dnsmap.googlecode.com'
-license=('GPL')
-makedepends=('make')
+groups=('blackarch' 'blackarch-fingerprint')
+url='https://github.com/resurrecting-open-source-projects/dnsmap'
+license=('GPL3')
+makedepends=('git' 'make')
```

```suggestion
makedepends=('git')
```

- https://wiki.archlinux.org/title/PKGBUILD#depends
- https://archlinux.org/packages/core/any/base-devel/